capstone/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+# Uncomment this line and insert your own json key file for local testing
+spring.cloud.gcp.storage.credentials.location=file:/home/jennysheng/service-account.json
```

Comment:
Wouldn't this configuration also cause the application to look for the file when deployed to App Engine?
And it won't find it.

Reply:
+1; definitely comment it out when committing.